src/go/k8s/apis/redpanda/v1alpha1/console_types.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// ConsoleSpec defines the desired state of Console
+type ConsoleSpec struct {
+	// +optional
+	// +kubebuilder:default=console
+	MetricsNamespace string `json:"metricsNamespace"`
```

Comment:
is this kubernetes namespace or ... ? I assume not? Can we find better name than namespace?

Reply:
It's a prefix for exported metrics [REF](https://github.com/redpanda-data/console/blob/master/docs/config/console.yaml#L183)

Reply:
Marking as resolved [REF](https://github.com/redpanda-data/redpanda/pull/5758/commits/3bebfb72f71a19ac147351b72ee853e0b44e7b3f)